### ISSUE TITLE
UX(UI) change of the map

### ISSUE THREAD

**Issue by sebastiangaertner** [Greenstand/treetracker-web-map-client#1721]
Since the map section of the application seems to always have a performance issue and users are getting repelled by waiting too long we have the suggestion to move the focus to the individual tree and especially in the mobile view to show the project/planter/tree-slider first  before the map. 
Additionally the map should zoom in all the way into one specific "latest or youngest" capture. That way we have no delays and the mind of the user can focus on available info rather than waiting for a map to maybe load within the next minutes.

**Comment by dadiorchen** [comment]
@sebastiangaertner you mean cancel the flying animation on the map and directly jump to the final zoom level?

**Comment by sebastiangaertner** [comment]
@dadiorchen yes correct. We seem to not be able to create a performant experience so rather get the focus on an individual Capture.

**Comment by dadiorchen** [comment]
I will close this one a create a new one for this goal

**Comment by dadiorchen** [comment]
#1723
